extensions/amp-list/0.1/amp-list.js

```diff
@@ -892,6 +893,17 @@ export class AmpList extends AMP.BaseElement {
       });
   }
 
+  /**
+   * Builds an elided, shortened URL suitable for display in error messages from
+   * the given request.
+   * @param {!FetchRequestDef} request
+   * @return {string}
+   */
+  buildElidedUrl_(request) {
```

Comment:
If the desire is to just remove `__amp_source_origin`, why is this reducing to just origin? isn't the rest helpful for debug purposes (pathname, query params, etc)

Reply:
The desire is to be consistent with how amp-list behaves in the non-SSR path, which currently reduces it to just the origin.